erpnext/stock/doctype/item_price/item_price.json

```diff
@@ -33,7 +33,8 @@
   "valid_upto",
   "section_break_24",
   "note",
-  "reference"
+  "reference",
+  "update_existing_price_list_rate"
```

Comment:
This should be a global setting in "Stock Settings" instead of on individual item price records. 

It should be next to "auto_insert_price_list_rate_if_missing" and unchecked by default.